src/guide/blockchain/metadata.md

```diff
@@ -0,0 +1,162 @@
+# Metadata
+
+Metadata are key-value pairs that are attached to objects in the
+blockchain. The following can contain metadata:
+
+- domains
+- accounts
+- assets
```

Comment:
Only store assets

Reply:
Yes, but it's not exactly what attached to the main object